server/conf/application.conf

```diff
@@ -14,6 +14,13 @@ play.crypto.secret = "changeme"
 # ~~~~~
 play.i18n.langs = [ "en" ]
 
+jira.baseUrl = "https://jira.agiledigital.com.au"
```

Comment:
Correct, but the API endpoints do not change, so we don't need to make them configurable.